ApartApps.md

```diff
@@ -64,6 +64,63 @@ Data specifically intended for the apartment application is stored in the CCT da
 The details of these tables can be viewed and edited in SQL Server Management Studio with the right permissions. 
 Although, a faster method for simply viewing the tables is to open `Gordon360/Models/CCT_DB_Models.edmx` in Visual Studio 2017 and pan the screen around until the box for the desired table is in view. In the edmx, the type and nullability of each attribute of a table can also be viewed by clicking a column name and looking at the Properties table in the bottom right corner of Visual Studio 2017.
 
+#### Example:
+
+AA_Admin
+- AdminID - varchar(10)
+
+| AdminID  |
+|----------|
+|222222222 |
+|333333333 |
+
+AA_ApartmentApplication
+- AprtAppID - int, not nullable
+- DateSubmitted - datetime, nullable
+- DateModified - datetime, not nullable
+- EditorID - varchar(10), not nullable
+
+| AprtAppID | DateSubmitted           | DateModified            | EditorID  |
+|-----------|-------------------------|-------------------------|-----------|
+| 1         | NULL                    | 2021-01-01 12:12:12.111 | 123456789 |
+| 2         | 2021-01-01 12:12:12.111 | 2021-01-01 12:12:12.111 | 111111111 |
+
+AA_ApartmentChoices
+- AprtAppID - int, not nullable
+- Ranking - int, not nullable
+- BLDG_CDE - char(5), not nullable
+
+| AprtAppID | Ranking | BLDG_CDE |
+|-----------|---------|----------|
+| 1         | 1       | TAV      |
+| 1         | 2       | BRO      |
+| 2         | 1       | TAV      |
+| 2         | 2       | BRO      |
```

Comment:
I thought the current plan was to change the column name and make the values match what we see in AA_ApartmentHalls?
```suggestion
| AprtAppID | Ranking | Hall |
|-----------|---------|----------|
| 1         | 1       | Tavilla      |
| 1         | 2       | Bromley      |
| 2         | 1       | Tavilla      |
| 2         | 2       | Bromley      |
```

Reply:
Correct, `Hall` or `HallName`? I personally prefer `HallName` for this case, it emphasizes that this is a string of the name

We also need to update the data type of this to match the length of the `Name` from `AA_ApartmentHalls`

Reply:
Yes, the datatype needs to be changed. `HallName` sounds good to me. 
```suggestion
| AprtAppID | Ranking | HallName |
|-----------|---------|----------|
| 1         | 1       | Tavilla      |
| 1         | 2       | Bromley      |
| 2         | 1       | Tavilla      |
| 2         | 2       | Bromley      |
```

Reply:
Good catch. I just copied it from the database table. HallName makes more sense, I agree. We probably should make a note on the project board to update the database table attributes.